pin_packing.graphql

```diff
@@ -0,0 +1,158 @@
+
```

Comment:
We don't want to commit the graphql schema, but we should come up with some way of generating it ahead of time...